equinix/provider.go

```diff
@@ -249,51 +248,3 @@ func isEmpty(v interface{}) bool {
 		return false
 	}
 }
-
-func slicesMatch(s1, s2 []string) bool {
```

Comment:
This was only used in acceptance tests, so it has moved to the abomination that is the `helper_test.go` file